### DISCUSSION TITLE
Laravel 8

### DISCUSSION THREAD

**Discussion by dwiyudirayia** [spatie/laravel-medialibrary#2050] [Q&A] [answered]
Is this package support laravel 8? I was upgrade my project to laravel 8 and get error. Please help me :(

**Comment by freekmurze** [comment, marked as answer]
Yes, it is supported.

https://github.com/spatie/laravel-medialibrary/blob/8.7.1/composer.json#L28-L32

**Reply by dwiyudirayia** [reply]
Okay thanks sir. Just reisntall this package and BOOM this package support for laravel 8
